activesupport/lib/active_support/callbacks.rb

```diff
@@ -62,8 +62,7 @@ module Callbacks
 
     included do
       extend ActiveSupport::DescendantsTracker
-      class_attribute :__callbacks, instance_writer: false
-      self.__callbacks ||= {}
```

Comment:
Same issue here: "Probably shouldn't replace this with a default:, right? Given that it checks the parent class first."